fix return value of "kubectl auth cani --quiet"

CaoShuFeng opened the pull request:
Before this change:
$ kubectl auth can-i get pod
no
$ kubectl auth can-i get pod --quiet
$ echo $?
0

After this change:
$ kubectl auth can-i get pod
no
$ kubectl auth can-i get pod --quiet
$ echo $?
1

@deads2k 

**Release note**:
<!--  Steps to write your release note:
1. Use the release-note-* labels to set the release note state (if you have access) 
2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`. 
-->
```NONE
```

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/43900)
<!-- Reviewable:end -->

CaoShuFeng committed:
fix return value of "kubectl auth cani --quiet"

Before this change:
$ kubectl auth can-i get pod
no
$ kubectl auth can-i get pod --quiet
$ echo $?
0

After this change:
$ kubectl auth can-i get pod
no
$ kubectl auth can-i get pod --quiet
$ echo $?
1

deads2k commented:
/lgtm
/release-note-none

deads2k commented:
@fabianofranz do you know the process for a backport to 1.6.1?

deads2k commented:
@k8s-bot kubemark e2e test this

fabianofranz commented:
> do you know the process for a backport to 1.6.1?

@deads2k afaik it's in 
https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md

CaoShuFeng commented:
/cherrypick-candidate

deads2k commented:
/lgtm
